### PR TITLE
feat(sentry): use AppVersion as the default image tag

### PR DESCRIPTION
**Pull request by yoan-adfinis**
Make it easier to avoid out of sync versions in the future (with 20.7.2 using `AppVersion` should just be fine, #137)

### CODE REVIEW

File: sentry/templates/_helper.tpl [hunk 1]
```diff
@@ -12,6 +12,22 @@
...
+{{- end -}}
+{{- define "sentry.image" -}}
+{{- default "getsentry/sentry" .Values.images.sentry.repository -}}
+:
+{{- default .Chart.AppVersion .Values.images.sentry.tag -}}
+{{- end -}}
+{{- define "snuba.image" -}}
+{{- default "getsentry/snuba" .Values.images.snuba.repository -}}
+:
+{{- default .Chart.AppVersion .Values.images.snuba.tag -}}
```
Comment (DandyDeveloper) [inline sentry/templates/_helper.tpl:28]: Shouldn't this be a `tpl` with the context passed in? `tpl .Chart.AppVersion .`?

**Review by DandyDeveloper — COMMENTED** [review]
I'd need to triple check, but doesn't refs like this need to be a tpl in the templating?

File: sentry/templates/_helper.tpl (hunk 1 above)
Comment (yoan-adfinis) [reply sentry/templates/_helper.tpl:28]: It seems to work as is.

```console
$ helm install --dry-run --generate-name --debug . | grep "getsentry\\/"
install.go:159: [debug] Original chart version: ""
install.go:176: [debug] CHART PATH: /home/yoan/soft/charts/sentry

        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
          image: "getsentry/relay:20.7.2"
        image: "getsentry/relay:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/sentry:20.7.2"
        image: "getsentry/snuba:20.7.2"
        image: "getsentry/snuba:20.7.2"
        image: "getsentry/snuba:20.7.2"
        image: "getsentry/snuba:20.7.2"
```

Comment (DandyDeveloper) [reply sentry/templates/_helper.tpl:28]: Interesting! I also gave bad code haha, but good to know.

**Comment by Mokto** [comment]
Thanks! I think this is ready for merging, right?

**Comment by hairmare** [comment]
> Thanks! I think this is ready for merging, right?

I just added the template calls to all the new deployments and hooks from 5.1.0, so yeah I think so.

**Comment by Mokto** [comment]
Ah, good catch! Merging now

**Pull request merged**